src/components/guidance-content/info/InformationItems.js

```diff
@@ -0,0 +1,617 @@
+// Central resource for LabelWithInfo and Information items used throughout
+// the app.
+
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Button, Glyphicon, Table } from 'react-bootstrap';
+import LabelWithInfo from '../../guidance-tools/LabelWithInfo';
+import Information from '../../guidance-tools/Information';
+import css from './InformationItems.css';
+
+///////////////////////////////
+// Parts for building items
+///////////////////////////////
+
+
+///////////////////////////////
+// Contact info
+///////////////////////////////
+
+export const appContact = {
+  email: 'mailto:rglover@uvic.ca',
+  name: 'Rod Glover',
+};
+
+
+///////////////////////////////
+// Dataset filters (Model, Emission, Variable)
+///////////////////////////////
+
+export const datasetFilterPanelLabel = (
+  <LabelWithInfo label='Dataset Filter'>
+    <p>
+      This is the primary filter for the datasets to be displayed.
```

Comment:
Maybe switch "This is the primary filter for the datasets to be displayed" to "This is the primary filter used to select datasets to display"? The "for" seems unclear as to what the relationship between filter and datasets is.